web/src/locales/sv.json

```
@@ -75,15 +98,165 @@
       "total-electricity-consumption": "Total ekonsumtion per källa"
     }
   },
+  "feedback-card": {
+    "title": "Hjälp oss att förbättra!",
+    "success-message": "Ditt svar hjälper oss att förstå och förbättra kvaliteten på vår app.",
+    "success-subtitle": "Tack för din återkoppling!",
+    "estimations": {
+      "subtitle": "Vänligen betygsätt följande påstående.",
+      "secondary-question": "Finns det något vi kan göra för att förbättra det?",
+      "primary-question": "Beskrivningen av dataestimeringen var lätt att förstå:"
+    },
+
+    "primary-question": "Hur nöjd är du med din upplevelse av att använda Electricity Maps-appen?",
+    "secondary-question-low": "Vad kan vi göra bättre?",
+    "secondary-question-high": "Vilka förbättringar skulle du vilja se härnäst?",
+
+    "optional": "Valfritt:",
+    "input-question": "Berätta vilka finesser du skulle vilja se?",
+    "agree": "Instämmer helt",
+    "disagree": "Instämmer inte alls",
+    "placeholder": "Berätta mer med några ord...",
+    "submit": "Skicka"
+  },
+  "estimation-badge": {
+    "partially-estimated": "Delvis uppskattad",
+    "fully-estimated": "Uppskattad",
+    "outage": "Otillgänglig"
+  },
+  "estimation-card": {
+    "link": "Läs om våra estimeringsmodeller",
```

```suggestion
    "link": "Läs om våra uppskattningsmodeller",
```
If we are using uppskattning for estimations we should probably use it here too to stay consistent.